src/layout/Header/index.tsx

```diff
@@ -13,60 +14,95 @@ const Header = () => {
   const handleMouseLeave = () => {
     setIsHovered(false);
   };
+
+  const handleMouseClickOnMenuButton = () => {
+    setIsMenuShown(!isMenuShown);
+  };
+
   return (
-    <header className={styles.header}>
-      <div className={styles.content}>
-        <nav className={styles.nav}>
-          <h1 className={styles.nav__heading}>TaoTalk</h1>
-          <div className={styles.nav__divide}>|</div>
-          <ul className={styles.nav__list}>
-            <li>
-              <a title="Home" href="#" className={styles.nav__list_item}>
-                Home
-              </a>
-            </li>
-            <li>
-              <a
-                title="Textbook"
-                href="#"
-                className={[styles.nav__list_item, styles['nav__list_item--inactive']].join(' ')}
-              >
-                Textbook
-              </a>
-            </li>
-            <li>
-              <a
-                title="Statistics"
-                href="#"
-                className={[styles.nav__list_item, styles['nav__list_item--inactive']].join(' ')}
-              >
-                Statistics
-              </a>
-            </li>
-            <li
-              className="relative"
-              onMouseEnter={handleMouseEnter}
-              onMouseLeave={handleMouseLeave}
-            >
-              <a
-                title="Games"
-                href="#"
-                className={[styles.nav__list_item, styles['nav__list_item--inactive']].join(' ')}
-              >
-                Games
-                <img src="/down_arrow.svg" alt="down array"></img>
-              </a>
-              {isHovered && (
-                <div className=" absolute top-[46px] w-[215px] rounded-[14px] bg-white shadow-[0_343px_343px_0_rgba(23,64,51,0.08)] ">
-                  <DropDownMenu menuContent={['Sprint →', 'Audio-call →']} />
-                </div>
-              )}
+    <div>
+      <header className="flex h-[88px] items-center justify-center bg-tk_greyLight shadow-xl">
```

Comment:
https://github.com/taodemy/taotalk/blob/d8f609ef26d9a3dc290c100b53baff68a5dfc43a/src/layout/Header/index.tsx#L24

1. Box shadow has been reset to be the same as in figma. In my previous trials, I forgot to set "inset" in shadow settings, so it cannot work properly and I use shadow-xl instead. This time I searched on the Internet to know how to set box shadow in traditional CSS and tailwind. 

2. Heigh has been set to `h-[72px] md:h-[88px]`. Thank you for your notice, otherwise I cannot observe the different height setting in responsive design.